docs/_documentations/mdt-vsc-importedprojects.md

```diff
@@ -1,30 +1,30 @@
 ---
 layout: docs
-title: Adding projects
+title: Adding existing projects
 description: Documents
-keywords: importing, directory, archive, configuring, cloud, microservices, application, Eclipse, MicroProfile, Java, Spring, Node.js, Swift, Maven, Dockerfile, GitHub, container, Liberty, Helm, Dockerfile-lang, local directory, import, pre-import, pom.xml, server.xml, Maven, Docker, Helm chart
+keywords: importing, directory, archive, configuring, cloud, microservices, application, Eclipse, MicroProfile, Java, Spring, Node.js, Swift, Maven, Dockerfile, GitHub, container, Liberty, Helm, Dockerfile-lang, local directory, add existing project, pre-import, pom.xml, server.xml, Maven, Docker, Helm chart
 duration: 1 minute
-permalink: importedprojects
+permalink: mdt-vsc-importedprojects
 type: document
 parent: importingaproject
 order: 0
 ---
 
-# Adding projects
+# Adding existing projects
```

Comment:
Two things:

1) This appears to be a duplicate of the Eclipse page. Does it make sense to put this in a common place?
2) Is there anything here which actually directs the user to the "Add Existing Project" command in VS Code?

Reply:
1. There are a couple of duplicates of topics for now, this is because of some strange behaviour in the UI that if you locate the same topic in two places, when you open one of them, it opens both in the toc which is confusing for our users. 
2. There is the Adding existing projects topic in the VS Code section, but this will probably be reworked, and I'll make a note of it, in Rework the Codewind for VS Code section, issue #45

Reply:
Right, I just mean we should explicitly tell them to right-click `Projects` in the Codewind tree and click `Add Existing Project`. Or click `Add Existing Project` in the Command Palette.

Reply:
Hi Tim, Looking again this morning, that info's in the `Getting started` for Eclipse & VS Code, for example, https://github.com/eclipse/codewind-docs/blob/master/docs/_documentations/mdt-vsc-getting-started.md and https://github.com/eclipse/codewind-docs/blob/master/docs/_documentations/mdt-eclipse-getting-started.md so we're covered.